docs/latest/custom-tools/event-dispatcher-callbacks.md

```diff
@@ -1,3 +1,35 @@
 ## Event Dispatcher Callbacks {#event-dispatcher-callbacks}
 
-_This section needs content_
+Here we can add the meat of our tool. Event dispatchers check for methods on tools and fire them when appropriate [TODO: List them all?? Or just link to api? Not sure yet. ]
```

Comment:
An image that describing event order, how tool's are chosen to be dispatched to, the methods that are associated with each event, and  `isToolLocked` would be super handy here. I'll look in to generating something for this?

Reply:
That would honestly be super neat.

Reply:
Created a note here: https://github.com/cornerstonejs/cornerstoneTools/projects/2#card-13374470

Feel free to do the same for anything you want to pass off to someone else or make a note of. Comments can be easily forgotten after a merge.

Reply:
Sure, I more wrote that comment to myself haha, planned to continue this morning. But yeah I'll post any discussion/issues back to the board.